CI/vale/vale_styles/Vocab/Besu/accept.txt

```diff
@@ -68,6 +68,7 @@ JMeter
 jnacl
 Kafka
 kalium
+Keycloak
```

Comment:
Not sure what this is but I assume this is meant to be in here

Reply:
It's because this name is otherwise not valid per our spellchecker.